omero/figure/index.html

```diff
@@ -22,7 +22,7 @@ <h1 class="hero-main-header"><img src="{{ site.baseurl }}/img/logos/omero-figure
                     <h4>Create quality figures<br>from your OMERO images</h4>
                     <br>
                     <a href="https://pypi.python.org/pypi/omero-figure/" class="large btn-red button sites-button" style="text-shadow: none;">Install OMERO.<span class="lowercase">f</span>igure</a>
-                    <a href="{{ site.baseurl }}/2020/05/25/figure-4-3-1.html" class="hero-link-dark" style="text-shadow: none;">Read the Release Announcement</a>
+                    <a href="https://github.com/ome/omero-figure/blob/master/CHANGELOG.md" class="hero-link-dark" style="text-shadow: none;">See changes</a>
```

Comment:
Maybe "See latest changes" would make the link seem more relevant?

Reply:
what do you suggest? This is what we have on the iviewer page

Reply:
Ah - I see you have "See changes" for iviewer too. Maybe update that too if you think "latest" is better? Otherwise np.

Reply:
Sorry - posts crossed at the same time there - I was suggesting just use "See latest changes" instead of "See changes" on figure and iviewer pages.

Reply:
done